docs/api/remix.md

```diff
@@ -98,10 +98,10 @@ import { NavLink } from "remix";
 function NavList() {
   // This styling will be applied to a <NavLink> when the
   // route that it links to is currently selected.
-  let activeStyle = {
+  const activeStyle = {
```

Comment:
This should be a `let`. Let's remove `prefer-const` from the docs dir and just use one consistent style throughout the whole repo.

Reply:
This should be reverted. Reminder that we'll handle all the stuff in the docs ahead of the next release when we merge everything from `main`.

Reply:
@mjackson Does this mean we don't want any user-facing code (so templates + docs) to use `const` anymore (like suggested/introduced by @kentcdodds a couple of weeks/months(?) ago) and _always_ use `let`?